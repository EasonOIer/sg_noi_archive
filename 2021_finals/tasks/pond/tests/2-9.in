1931 974
999699 999188 998085 996503 995557 994374 994293 993470 990326 990228 989402 988772 986770 982071 980737 979133 976846 975255 973694 972692 970808 970488 970232 970116 968783 968060 967952 967289 965115 964928 964235 963523 961975 960733 959916 957674 957181 955502 955433 953639 952333 952312 951437 950371 949792 949678 949061 948270 947184 946185 945602 943797 942069 941511 940938 940279 939528 934974 934019 932994 932857 931059 930335 930269 930061 929267 927882 927751 927520 926219 921946 921305 921213 921169 920610 919306 918469 917464 916069 915372 915042 914904 914750 914639 911750 908931 906387 906074 905374 903536 902348 901529 900569 898869 898044 895931 895160 893326 891693 889649 889617 884858 884281 883027 882896 882485 881076 879748 878056 875927 875093 873648 873422 873098 872843 872517 871947 870381 869120 867975 867083 866759 865228 864102 862367 861765 860769 860353 858415 857522 853958 853561 849485 846786 846038 846030 845475 845150 843094 842929 842835 841481 839647 839617 838232 837349 836842 836746 836608 835916 835354 834258 831308 830051 829439 828383 827280 824805 824771 822131 818726 818556 818153 817526 816489 816456 815869 815530 813874 808845 808752 808110 807782 807476 806847 805444 805298 804933 804784 802971 802403 801287 801062 799500 798162 796506 794824 793350 789531 788551 788114 788067 787919 787013 785103 785031 784962 783406 783233 782182 780226 779189 777086 776567 774521 773746 773446 772436 770940 769970 769795 768234 766735 765751 765441 763816 763093 762241 759605 758824 757764 757299 755588 754791 754508 753706 753132 750058 749542 749396 748042 745811 745410 745408 743430 742117 741407 740322 739637 736724 735755 735153 733108 731480 730556 730097 729786 729752 728096 727114 725876 724840 724256 723902 722527 719692 718784 718221 718052 717953 717860 717271 715733 715605 712417 712404 712016 709553 708446 708430 708093 705598 705528 705025 703664 703325 702858 701588 701354 699531 698505 697862 697549 697529 694859 693650 693425 693005 692222 691297 691241 688360 687717 687443 686346 685781 684555 684313 684055 683667 683424 681729 680826 680037 679528 678915 678128 677867 674387 673280 668346 666806 663736 659610 658759 658643 658052 658052 657224 656521 656374 656215 655290 654299 651241 650081 649681 649413 649402 647416 645672 643729 642870 642279 641977 641329 638331 637584 636488 636419 634948 634342 634128 633534 632994 632985 629784 629740 628967 628338 627850 626503 624759 624030 620755 619444 619084 618688 618127 618072 615706 614904 614728 614191 613385 613078 612646 611648 609216 609163 608829 608798 608797 607241 606553 603433 603397 602425 601776 601561 601521 600679 600079 599260 598904 597497 596782 596727 596727 596712 596606 596502 595844 595204 594190 593935 591713 589930 589179 588780 588034 586657 585661 584536 584210 582115 581897 581405 580950 580905 580765 579969 579325 575694 573244 571944 570270 569574 567483 567164 565166 562777 560254 558105 556380 556238 555841 555439 553895 553653 552130 551225 551197 550842 550637 549946 549836 549043 546561 545064 544284 542444 542055 541137 540521 538897 537945 534606 534238 534134 533652 533340 533256 532637 528855 528764 528146 526029 525829 525365 522172 520908 520716 518694 518565 518251 516000 515150 514568 512540 511189 510955 509105 508833 508444 508402 508007 507532 505999 505937 505166 504401 502869 502258 501428 500725 499373 499299 498199 497692 496428 496234 496004 495814 494824 494393 493376 490614 490558 489970 489088 488981 488458 487240 486158 486128 485815 485805 484498 484490 484391 484040 482333 481578 481285 480047 479692 477398 477006 476100 474999 474497 474100 470072 469708 469570 469569 468736 467780 467188 466975 464419 463914 462313 461621 461272 459018 457960 457132 454888 454485 452581 451140 449095 447456 444429 443855 443783 443693 440971 438502 438293 436334 435976 435152 434259 431251 430692 430610 430481 428419 424674 423268 422687 420893 420615 420282 418246 418003 416412 413835 412763 411409 411118 410758 410530 410056 409303 407327 406847 403849 403234 402995 402776 401482 401381 400044 399287 399105 396227 394908 393895 393489 393321 393062 393055 392030 391208 388892 387874 387223 385843 383902 383599 381719 381348 381072 380179 380105 376909 376014 374512 370560 370343 368639 368526 367610 366660 365097 364501 363221 360572 357714 357485 352725 352504 351724 349200 345326 345034 344992 344276 341283 341064 339333 339024 334513 332205 331203 330613 329003 328721 328487 327418 326041 324800 324766 321024 319784 319241 318976 318699 318102 317467 317375 316052 314849 314817 313743 313582 313213 312949 311962 311401 311377 311214 310108 308924 307759 307462 304320 304295 301706 300426 300356 299830 299721 298449 296986 296863 295340 292782 291930 290112 290019 289521 289474 289457 289356 286864 286271 285859 283813 282435 281670 280055 279438 278763 276723 274680 273584 272984 272692 272137 272102 270306 270032 269548 268494 268223 267467 267153 265524 264134 263711 263560 261813 261385 260360 259271 258643 257762 257161 255052 254682 253825 253251 253160 252786 247951 245625 244862 244180 243917 243719 242910 241637 240439 237895 237207 236883 236610 231531 231018 229721 227656 227484 226824 225693 224311 223723 221782 219344 218257 217966 215721 212800 210747 210624 208890 208443 208218 207999 207983 207145 206995 205814 203709 200660 199382 199030 198435 198285 198191 197608 197560 196858 194983 190990 189946 189936 189041 187624 185485 185217 184766 184734 184124 181806 181320 177839 177183 172454 171924 169700 169038 168189 167722 166764 166326 165400 161631 160260 158786 158180 156407 153909 153788 151955 151012 150673 149757 146978 146209 146017 145324 144184 142800 142406 140419 138744 137572 137393 137015 136938 135701 131734 130796 130212 130104 129492 129213 128934 127584 127106 126897 124999 124723 122143 121214 120765 119910 119691 119610 118793 118664 118559 117556 116487 116225 115329 114425 114362 113688 113317 112663 111907 110780 109997 109713 109274 104299 103193 103083 103062 101151 100876 100183 99388 99294 98936 98690 98660 98165 97697 97440 96282 96127 95592 95029 92538 91977 86606 83987 83299 81803 81346 80946 80542 80294 79802 79239 78615 78146 77969 77487 76970 76622 76558 75508 71457 68095 67192 66706 64756 63875 62568 62282 61326 60189 58842 58662 58003 56592 55541 54895 53729 53570 52058 51773 51134 50653 50098 48763 48410 48051 47706 47205 47105 45659 44377 44224 42468 40992 40242 38422 37544 36992 35966 33444 32625 30025 29634 28941 28853 27794 27683 26240 25406 23050 19577 18743 18589 17488 17094 16378 15781 15222 13445 10666 10429 9566 9048 8828 7586 6670 6576 4313 3216 2543 2338 1548 337 653 1803 2556 5835 7362 8610 8717 11527 11616 12372 12484 12905 13367 13648 13783 14208 15564 15703 18054 18085 18111 19943 21956 22502 23389 23681 23815 24069 24307 24664 24956 26607 27480 27860 28510 30213 33079 33149 35506 35601 35620 37134 37628 37991 38739 38873 39680 40830 41427 42165 42797 43359 43714 45538 45652 46192 46810 48873 49181 49580 50780 51985 52011 52517 52752 55135 55537 56013 56609 56897 56939 57154 57983 61155 61346 63297 65229 65720 70400 71272 71346 73135 73979 74870 75769 78798 79402 79527 79977 80949 80966 81530 84008 84064 86837 87384 87411 88859 91448 95088 95308 96069 96327 96454 98760 98997 99360 101062 101134 102437 104928 105451 105752 107114 107838 108244 109580 110117 111885 114455 114540 115000 115482 115511 115617 117109 117412 118557 120109 120163 120567 121929 122515 122977 125301 125876 126581 128095 128116 128452 128686 129035 129870 131993 134614 134869 136023 136413 137165 141880 142558 142747 143010 144808 146158 147693 149189 150168 150917 152935 153270 153468 153883 154188 154459 160504 162177 162944 163949 165872 166019 166047 166574 167475 168354 168823 168891 170367 171544 172145 174118 174141 174784 175255 175523 175590 176570 177029 177907 179201 179904 180691 182290 183209 185209 185806 187044 187345 187458 192689 193165 194426 197694 199977 200351 200610 201464 201508 202184 202444 203118 206188 207434 209822 210113 211328 214042 214160 214321 217362 217400 217762 218231 219039 221011 221092 223044 224221 224504 225253 226916 228189 228362 228457 230143 232902 234560 235328 236254 236424 236854 237606 238129 240384 240770 241806 242988 243195 244663 247364 249923 252502 253418 254673 255116 255309 255560 258450 258550 258571 258862 260220 262077 262169 263246 264867 268197 268798 269369 274676 274828 275981 276566 279102 279438 280518 280784 281361 283482 284453 285999 289430 290646 291937 294078 294568 296517 296586 298133 298862 298872 300162 300984 302610 303001 303395 303945 305552 307316 307322 310247 310806 310954 312256 313500 315406 318647 319047 320670 323605 323817 325515 325951 326056 326591 327297 327708 328028 328525 329995 330942 331146 332201 333220 334054 335257 336453 337443 338431 338978 340559 341194 343544 344878 344938 345333 346217 348564 348954 349098 351749 353614 353843 354473 355825 356984 359130 359387 360803 361413 361458 361759 361885 362279 362566 363088 364093 364917 366028 367931 368787 370830 374826 375371 376157 376315 376617 377333 377417 378281 380503 382684 385554 385934 387768 387997 388194 388999 389001 389073 393330 393377 394053 394952 395414 396038 397505 399709 400896 401721 402278 402278 402349 404187 405210 407767 408004 408749 411909 413915 414167 416153 423281 425468 426420 427271 427455 427994 430664 430806 431425 431793 433900 434302 439737 441693 443462 443867 444470 444641 444765 447301 448657 448948 449156 451288 453017 453569 453728 457116 457356 459861 460723 462757 464058 464566 465579 465873 467640 467793 469514 469531 470659 471403 471925 473228 473470 474706 476287 481186 481605 481862 487287 487562 489200 489403 489780 489900 489994 490576 491022 492831 494723 495954 497606 501405 502864 503511 505428 506281 509238 511081 511924 512217 512772 513732 515363 516010 517657 517832 519652 519765 520786 522158 523050 523489 523895 524296 524698 526828 527247 527630 528283 530434 534160 534524 535469 535940 536385 536499 537876 539819 539889 540669 541715 542009 542911 543678 544724 546404 547007 547318 547965 548669 550210 552258 552359 555431 555595 558046 558184 559673 560053 560410 560686 561591 561747 561970 562043 562750 562924 563612 565293 565626 566507 568595 571256 573530 574306 574401 574658 574955 575080 575508 577100 577182 580617 580791 580901 582988 583782 584878 584957 586147 587271 587921 588255 589557 589874 590822 592001 593581 595873 596609 597335 598793 599135 601654 602476 605546 605727 607360 609580 610769 612589 614494 614508 616085 616134 616168 616538 618334 618992 620605 622397 622947 623293 623375 623717 626260 627394 627825 627871 629177 630270 630968 632690 633680 633928 633993 634581 635109 635984 636125 637371 637801 638093 638371 639945 641116 641655 642169 643878 646410 646476 646761 647816 648857 648900 649581 649770 651042 652155 652240 652272 652567 652807 654479 654598 654971 655859 655959 656874 657084 657257 657414 657692 658050 658198 658881 659954 660934 661436 661967 662238 666789 667819 668353 672065 677737 678151 678367 680530 682073 682416 683588 683820 684731 687596 688570 689022 689063 689302 689418 689596 690698 690703 690896 692660 693376 695684 697554 698053 701725 703689 705174 705632 706312 706424 706443 707801 707845 709075 709911 711125 712577 712828 713845 714608 717151 718320 720902 721698 722338 723975 724087 726321 726871 727989 729461 731039 732332 732940 733055 733171 736412 736959 737484 738014 738967 741008 741471 742580 746101 747006 747087 748461 748778 751983 753008 755753 756672 757468 757629 760231 760709 761484 762751 762873 763216 764903 765419 765737 765849 766726 767985 769417 773029 775722 775969 776524 776799 777204 777441 777684 778062 779189 781715 783101 783480 784929 785087 788428 789411 789488 790044 791102 792091 793829 796941 797536 800224 802659 802939 802998 803427 805304 805828 807789 813212 814457 814708 814979 815128 816134 818950 819872 820244 820759 822643 824502 824582 824644 824996 826927 831309 834254 837969 838263 839514 841100 841491 842027 842302 842337 845289 847751 848792 849031 849037 849747 849817 850002 851367 851392 852061 852985 853326 854441 854499 854948 855348 856182 857267 859001 861801 866636 866722 867117 867243 868330 868755 868781 869039 869394 869517 870235 872058 872306 874400 874681 875273 875985 876650 880835 881065 881550 882792 884251 884914 888239 894549 895767 896265 896978 898110 898749 898990 899428 900169 900473 900618 902952 903112 904729 905480 907038 907987 909462 909848 911070 912405 913068 913224 913761 915673 917181 917972 918930 921100 921208 922874 922958 924591 924913 925103 926040 926215 927904 928978 929218 929310 929408 929590 929748 930414 930850 931182 932509 933867 935080 935791 936377 938814 940321 941451 941738 942444 942857 942965 943360 945739 946285 946592 947078 948163 948794 948925 949153 949561 949713 950567 952134 952362 952949 954235 957323 958734 960071 962039 962418 962821 964006 965040 965470 967017 967390 967485 968646 969779 970221 971027 972125 972165 972170 972286 973038 975635 976361 981001 982732 983328 983424 985813 986931 988324 989546 990317 990888 991283 992153 992675 994655 995629 996201 997755 998764
