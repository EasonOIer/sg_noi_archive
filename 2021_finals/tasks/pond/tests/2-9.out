1078082954315